src/main/java/org/folio/fqm/service/EntityTypeService.java

```diff
@@ -56,17 +56,31 @@ public class EntityTypeService {
    * @param entityTypeIds If provided, only the entity types having the provided Ids will be included in the results
    */
   @Transactional(readOnly = true)
-  public List<EntityTypeSummary> getEntityTypeSummary(Set<UUID> entityTypeIds) {
+  public List<EntityTypeSummary> getEntityTypeSummary(Set<UUID> entityTypeIds, Boolean includeInaccessible) {
```

Comment:
Nit: I'd make this parameter a primitive `boolean` and do the unboxing in the controller. That way, it separates the concerns of the setting being optional from the business logic of actually handling the setting